.gitignore

```diff
@@ -39,3 +39,4 @@ publish_to_bower/
 bower_components/
 npm-debug.log
 .ember-cli
+/.idea
```

Comment:
This is generally better suited for a system wide `.gitignore` file as described in https://help.github.com/articles/ignoring-files/#create-a-global-gitignore.